runtime/gc_vlhgc/ContinuationObjectBufferVLHGC.cpp

```diff
@@ -107,3 +109,37 @@ MM_ContinuationObjectBufferVLHGC::addForOnlyCompactedRegion(MM_EnvironmentBase*
 		}
 	}
 }
+
+void
+MM_ContinuationObjectBufferVLHGC::iterateAllContinuationObjects(MM_EnvironmentBase *env)
+{
+	MM_GCExtensions *extensions = MM_GCExtensions::getExtensions(env);
+	MM_HeapRegionManager *regionManager = extensions->getHeap()->getHeapRegionManager();
+	MM_HeapRegionDescriptorVLHGC *region = NULL;
+	GC_HeapRegionIteratorVLHGC regionIterator(regionManager);
+
+	if (env->_currentTask->synchronizeGCThreadsAndReleaseSingleThread(env, UNIQUE_ID)) {
+		env->_currentTask->releaseSynchronizedGCThreads(env);
+	}
```

Comment:
The same as above - if sync point is necessary please use `MM_ParallelTask::synchronizeGCThreads()`, If not it can be just deleted

Reply:
same as above